src/utils/text.rs

```diff
@@ -9,18 +10,18 @@ pub fn align_text(text: &str, alignment: &str, maximum_length: u16) -> String {
         panic!("Parameter of 'maximum_length' cannot be below 1.");
     }
 
+    // Compute the display width of `text` with support of emojis and CJK characters
+    let mut dw = display_width(text);
+    if dw > maximum_length as usize {
+        dw = maximum_length as usize;
+    }
     match alignment {
-        "left" => text.to_string(),
-        "right" => format!(
-            "{text:>width$}",
-            text = text,
-            width = std::cmp::max(maximum_length, text.len() as u16) as usize
-        ),
-        "center" => format!(
-            "{text:^width$}",
-            text = text,
-            width = std::cmp::max(maximum_length, text.len() as u16) as usize
-        ),
+        "right" => format!("{}{}", " ".repeat(maximum_length as usize - dw), text),
+        "center" => {
+            let side_spaces =
+                " ".repeat(((maximum_length / 2) - (((dw / 2) as f32).floor() as u16)) as usize);
+            format!("{}{}{}", side_spaces, text, side_spaces)
+        }
```

Comment:
I was worried that this wasn't implemented before reviewing, thanks for realizing that this needed changes.

Reply:
I wanted a general solution, but this doesn't work with emotes(

Reply:
Unfortunate.